src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -346,6 +346,11 @@ internal static ImmutableArray<SyntaxReference> GetDeclaringSyntaxReferenceHelpe
         /// </summary>
         public abstract bool IsStatic { get; }
 
+        /// <summary>
+        /// Returns true if this symbol requires an instance reference as the implicit reciever. This is false if the symbol is static, or a <see cref="LocalFunctionSymbol"/>
+        /// </summary>
+        public virtual bool RequiresInstanceReciever => !IsStatic;
```

Comment:
>Reciever [](start = 44, length = 8)

Typo: Receiver

Reply:
I always get my "ei"s and "ie"s mixed up :-)

Anyway, fixed now.